textprocessors.cpp

```diff
@@ -24,8 +24,11 @@ void leet_case(char *text) {
 
 }
 
-void capitalize_letters(char *text) {
 
+void capitalize_letters(char * const text) {
```

Comment:
```suggestion
void capitalize_letters(char const * text) {
```
The issue with the **original** is that the values in the array are set to be constant, **not** the pointer. This should be the other way around.

Reply:
```suggestion
void capitalize_letters(char * text) {
```
Suddenly realizing **no const modifier can be used**. The pointer need to be moveable through the array, and the values need to be changeable.